### ISSUE TITLE
Fail gracefully if Punkt does not support the language

### ISSUE THREAD

**Issue by tuetschek** [GEM-benchmark/GEM-metrics#25]
For any data in a language not supported by Punkt, the tokenization now crashes. We should default to something.

**Comment by evanmiltenburg** [comment]
If we're sticking with the NLTK, then I'd suggest using either a whitespace tokenizer, or the `Toktoktokenizer` (because it has a funny name and [seems to work well)](https://www.nltk.org/api/nltk.tokenize.html#module-nltk.tokenize.toktok).

```
from nltk import ToktokTokenizer
toktok = ToktokTokenizer()
toktok.tokenize("Hello, world! This is a test.")
```

**Comment by tuetschek** [comment]
@evanmiltenburg : Hmm, you're suggesting ToktokTokenizer as a backup, not to replace Punkt, right?

I kind a like it, but there's one issue with full stops – it'll only split them from the preceding word at the end of the input, otherwise it'll assume it's an abbreviation. That means that you should always input 1 sentence only. I don't think this is satisfied...

I have a dumb language-indepedent (kinda, there is some English-specific stuff, but not much) [tokenizer of my own](https://github.com/tuetschek/futil/blob/master/futil/tokenize.py) that doesn't make the assumption, so maybe I'll use that instead – OK for you?

**Comment by evanmiltenburg** [comment]
Good point, we need to think of different kinds of texts. The 'dumb' tokenizer looks fine for now. If we can find a better one we can just replace it.

**Comment by tuetschek** [comment]
OK, fixed in 23cf44ea0c89cb54e3c8e41be424c81b8bcf90ba.
